composer.json

```diff
@@ -65,10 +64,10 @@
                 "Pagination does not work correctly for comment fields that are rendered using #lazy_builder": "https://www.drupal.org/files/issues/2020-12-22/pagination-does-not-work-with-lazy-builder-3189538-2.patch",
                 "Providing default route value for entity forms is not possible": "https://www.drupal.org/files/issues/2020-12-29/2921093-18.patch",
                 "Selecting the same day in a date between filter returns no results": "https://www.drupal.org/files/issues/2020-07-06/2842409-15.patch",
-                "Empty language (langcode) field wrapper inserted into contact form": "https://www.drupal.org/files/issues/2021-11-04/empty-langcode-field-wrapper-2842405-20.patch",
+                "Empty language (langcode) field wrapper inserted into contact form": "https://www.drupal.org/files/issues/2022-01-29/empty-langcode-field-wrapper-2842405-23.patch",
                 "Broken title in modal dialog when title is a render array": "https://www.drupal.org/files/issues/2019-10-21/2663316-76.drupal.Broken-title-in-modal-dialog-when-title-is-a-render-array.patch",
                 "Post update hook naming convention patch": "https://git.drupalcode.org/project/drupal/-/merge_requests/1215.patch",
-                "Flood MemoryBackend::events[] key of micro time cannot guarantee uniqueness": "https://www.drupal.org/files/issues/2021-12-12/2910000-33--floodmemorybackend-time-local.patch",
+                "Flood MemoryBackend::events[] key of micro time cannot guarantee uniqueness": "https://git.drupalcode.org/project/drupal/-/merge_requests/1451.diff",
```

Comment:
lets try to not use moving targets, if someone commits in your added MR it might break. Always create a local patch in the issue, or if you can't you can always open up an issue in Open Social and add a patch there for reference.

Reply:
Yes, I will upload a new patch here: https://www.drupal.org/project/social/issues/3253992